beater/config.go

```diff
@@ -107,15 +108,16 @@ func replaceVersion(pattern, version string) string {
 
 func defaultConfig(beatVersion string) *Config {
 	return &Config{
-		Host:               "localhost:8200",
-		MaxUnzippedSize:    50 * 1024 * 1024, // 50mb
-		MaxHeaderSize:      1 * 1024 * 1024,  // 1mb
-		ConcurrentRequests: 40,
-		ReadTimeout:        2 * time.Second,
-		WriteTimeout:       2 * time.Second,
-		ShutdownTimeout:    5 * time.Second,
-		SecretToken:        "",
-		AugmentEnabled:     true,
+		Host:                "localhost:8200",
+		MaxUnzippedSize:     30 * 1024 * 1024, // 50mb
+		MaxHeaderSize:       1 * 1024 * 1024,  // 1mb
+		ConcurrentRequests:  5,
+		MaxRequestQueueTime: 2 * time.Second,
```

Comment:
What's the logic behind this number?  This default seems very long.

Reply:
Mostly a gut call. With a `max_unzipped_size` of 50mb it could take a while to process incoming requests.